File: 프로그래머스/0/181904. 세로 읽기/세로 읽기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int m, int c) {
    string v;
    for(int i = 0; i < my_string.size(); i++) {
        if(i % m + 1 == c) v += my_string[i];
    }
    
    return v;
}

File: 프로그래머스/0/181904. 세로 읽기/세로 읽기.java
class Solution {
    public String solution(String my_string, int m, int c) {
        String v = "";
        for(int i = 0; i < my_string.length(); i++) {
            if(i % m + 1 == c) v += my_string.charAt(i);
        }
        
        return v;
    }
}